lib/src/scip_visitor.dart

```diff
@@ -84,6 +84,15 @@ class ScipVisitor extends GeneralizingAstVisitor {
   void _visitSimpleIdentifier(SimpleIdentifier node) {
     var element = node.staticElement;
 
+    // Both `.loadLibrary()`, and `.call()` are synthetic functions that
+    // have no definition. These should therefore should not be indexed.
+    if (element is FunctionElement && element.isSynthetic) {
+      if ([
+        FunctionElement.LOAD_LIBRARY_NAME,
+        FunctionElement.CALL_METHOD_NAME,
```

Comment:
I don't suppose the analyzer provides their own list of synthetic functions without definitions? :badpokerface:

Reply:
Unfortunately not

I pulled this directly from the analyzer code here: https://sourcegraph.com/github.com/dart-lang/sdk/-/blob/pkg/analyzer/lib/src/dart/analysis/index.dart?L145:27. There's also no references of [`LOAD_LIBRARY_NAME`](https://sourcegraph.com/search?q=context%3Aglobal+repo%3A%5Egithub%5C.com%2Fdart-lang%2Fsdk%24+FunctionElement.LOAD_LIBRARY_NAME&patternType=standard&sm=1&groupBy=path)